java/client/src/main/java/glide/api/models/BaseTransaction.java

```diff
@@ -1774,26 +1773,25 @@ public T bzmpop(double timeout, @NonNull String[] keys, @NonNull ScoreModifier m
 
     // TODO add @link to ZMPOP when implemented
     /**
-     * Blocks the connection until one or more elements, that are member-score pair, from the first
-     * non-empty sorted set in the provided list of key names.<br>
-     * <code>BZMPOP</code> is the blocking variant of <code>zmpop</code>.<br>
-     * The members with the highest/lowest score are popped from first sorted set that is non-empty,
-     * with the given <code>keys</code> being checked in the order that they are given.
+     * Blocks the connection unless it pops multiple elements, that is a member-score pair, from the
+     * first non-empty sorted set in the provided list of key names.<br>
+     * <code>BZMPOP</code> is the blocking variant of <code>ZMPOP</code>.<br>
+     * Members with the highest/lowest score are popped from first sorted set that is non-empty, with
+     * the given <code>keys</code> being checked in the order that they are given.<br>
      *
      * @see <a href="https://redis.io/commands/bzmpop/">redis.io</a> for more details.
      * @apiNote <code>BZMPOP</code> is a client blocking command, see <a
      *     href="https://github.com/aws/glide-for-redis/wiki/General-Concepts#blocking-commands">Blocking
      *     Commands</a> for more details and best practices.
-     * @param timeout The number of seconds to wait for a blocking <code>BZMPOP</code> operation to
-     *     complete. A value of <code>0</code> will block indefinitely.
+     * @param timeout The number of seconds to wait for a blocking operation to complete. A value of
+     *     <code>0</code> will block indefinitely.
      * @param keys The keys of the sorted sets.
      * @param modifier The element pop criteria.
      * @param count The amount elements to pop.
-     * @return Command Response - A two-element array with the first element being the name of the key
-     *     from which elements were popped, and the second element is a map of the popped elements.
-     *     Every entry in the map contains the popped member and its corresponding score.<br>
-     *     The data could be interpreted in format as <code>[key, Map{element: score}]</code>.<br>
-     *     If no element could be popped and the timeout expired, returns </code>null</code>.
+     * @return Command Response - A two-element <code>array</code> containing the key name of the set
+     *     from which elements were popped, and a member-score <code>Map</code>.<br>
+     *     The data could be interpreted in format as <code>[key, {element: score}]</code>.<br>
```

Comment:
This sentence still doesn't make sense grammatically. Do you mean: "The data could be interpreted as [key, {element: score}]"? It might also be easier to just delete this line entirely, since the above line already explains what is returned.

Reply:
I initially copied from another doc/PR. Removed.